pkg/cmd/image/convert.go

```diff
@@ -39,7 +39,6 @@ import (
 	"github.com/containerd/stargz-snapshotter/estargz"
 	estargzconvert "github.com/containerd/stargz-snapshotter/nativeconverter/estargz"
 	estargzexternaltocconvert "github.com/containerd/stargz-snapshotter/nativeconverter/estargz/externaltoc"
-	zstdchunkedconvert "github.com/containerd/stargz-snapshotter/nativeconverter/zstdchunked"
```

Comment:
Can you rather submit a PR to https://github.com/containerd/stargz-snapshotter ?

Reply:
Will see what I can do.